src/index.ts

```diff
@@ -42,18 +42,10 @@ async function main() {
   const client: MatrixClient = new MatrixClient(MATRIX_HOMESERVER_URL, MATRIX_ACCESS_TOKEN, storage, cryptoStore);
 
   // use puppeteer to bypass cloudflare (headful because of captchas)  
-  const chatGPT: ChatGPTAPIBrowser = new ChatGPTAPIBrowser({
-    email: OPENAI_EMAIL,
-    password: OPENAI_PASSWORD,
-    isGoogleLogin: (OPENAI_LOGIN_TYPE == "google"), 
-    isMicrosoftLogin: (OPENAI_LOGIN_TYPE == "microsoft"),
-    isProAccount: OPENAI_PRO
+  const chatGPT: ChatGPTAPI = new ChatGPTAPI({
+	  apiKey: OPENAI_API_KEY
```

Comment:
Another idea could be to introduce a new environment variable `OPENAI_MODEL` with a default of `text-chat-davinci-002-20221122` (as of today) so the users can change to model on their own as [shown here](https://github.com/transitive-bullshit/chatgpt-api/issues/292#issuecomment-1414304348).

Reply:
This sounds like the correct direction to go in. Thanks for all your work tracking these changes.

Reply:
Sure, I really enjoy talking to ChatGPT directly using my matrix client, so thank you for your working creating this bot.

I've just added a environment variable `CHATGPT_MODEL` as suggested